onetl/connection/db_connection/kafka.py

```diff
@@ -74,6 +76,16 @@ class Kafka(DBConnection):
         are derived from the Kerberos password. You can use this file to log on to Kerberos without being prompted for
         a password.
 
+    deploy_keytab : bool, default ``True``
+        If True Spark sends a keytab file to all executors. It is located in the root directory where the code is run.
+        Otherwise, the user is obliged to do this operation himself. On the Spark driver and executors,
+        the configs must be located in the same path.
```

Comment:
```suggestion
        If ``True``, Spark deploys a keytab file to all executors. It is stored in the current directory of both driver and executor.
        Otherwise, keytab file path should be absolute and exist on driver and all the executors.
```